content/applications/marketing/surveys.rst

```diff
@@ -15,6 +15,33 @@ shifting market trends.
 .. seealso::
    `Odoo Tutorials: Surveys <https://www.odoo.com/slides/surveys-62>`_
 
+.. cards::
+
+   .. card:: Create surveys
+      :target: surveys/create
+
+      Discover how to create surveys with Odoo.
+
+   .. card:: Scoring surveys
+      :target: surveys/scoring
+
+      Learn how to create and analyze survey scores with Odoo.
+
+   .. card:: Create questions
+      :target: surveys/questions
+
+      See how to create, configure, and customize all kinds of survey questions with Odoo.
+
+   .. card:: Live Session surveys
+      :target: surveys/live_session
+
+      Discover everything there is to know about Odoo's unique Live Session surveys.
```

Comment:
```suggestion
      Find out everything there is to know about Odoo's unique Live Session surveys.
```
Every card starts with a different word, but this is a repeat. Just trying to come up with a different one!